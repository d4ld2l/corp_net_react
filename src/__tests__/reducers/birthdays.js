import reducer from '../../redux/reducers/birthdays'

describe('birthdays reducer', () => {
  const initialState = {
    current: {},
  }

  it('should match GET_BIRTHDAYS_CURRENT_RES snapshot', () => {
    const action = {
      type: 'GET_BIRTHDAYS_CURRENT_RES',
      payload: {
        '2018-01-05': [
          {
            birthday: '1993-01-05',
            email: 'yuivannikov@phoenixit.ru',
            full_name: 'Иванников Юрий Юрьевич',
            id: 24,
            photo: {
              for_community: {
                url: '/uploads/profile/photo/24/for_community_Pickles.jpg',
              },
              for_profile: {
                url: '/uploads/profile/photo/24/for_profile_Pickles.jpg',
              },
              thumb: {
                url: '/uploads/profile/photo/24/thumb_Pickles.jpg',
              },
              url: '/uploads/profile/photo/24/Pickles.jpg',
            },
          },
        ],
      },
    }

    expect(reducer(initialState, action)).toMatchSnapshot()
  })
})
