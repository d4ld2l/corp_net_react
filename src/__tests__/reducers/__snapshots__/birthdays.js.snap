// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`birthdays reducer should match GET_BIRTHDAYS_CURRENT_RES snapshot 1`] = `
Object {
  "current": Object {
    "2018-01-05": Array [
      Object {
        "birthday": "1993-01-05",
        "email": "yuivannikov@phoenixit.ru",
        "full_name": "Иванников Юрий Юрьевич",
        "id": 24,
        "photo": Object {
          "for_community": Object {
            "url": "/uploads/profile/photo/24/for_community_Pickles.jpg",
          },
          "for_profile": Object {
            "url": "/uploads/profile/photo/24/for_profile_Pickles.jpg",
          },
          "thumb": Object {
            "url": "/uploads/profile/photo/24/thumb_Pickles.jpg",
          },
          "url": "/uploads/profile/photo/24/Pickles.jpg",
        },
      },
    ],
  },
}
`;
